tests/pre_merge/datasets/test_datamodule.py

```diff
@@ -165,6 +198,74 @@ def test_non_overlapping_splits(self, folder_data_module):
         ), "Found train and test split contamination"
 
 
+class TestAvenueDataModule:
```

Comment:
Since both these tests have the same code should we pass the fixtures as a list similar to what we do for model tests.?

Reply:
Aren't TestAvenueDataModule and TestUCSDpedDataModule basically the same?

Reply:
wow. I thought I removed those but apparently the changes were not pushed to Github.

Reply:
Fixed it now